hwbench/config/fio.conf

```diff
@@ -0,0 +1,18 @@
+# This configuration will :
+# - test /dev/nvme0n1 in 4k randread for 40 seconds
+# -- first with 4 stressors
+# -- then with 6 stressors
+#
+# As runtime is set to 30s by the user, it should be replaced by runtime=40 defined by hardware bench
+[global]
+runtime=40
+monitor=all
+
+[randread_cmdline]
+engine=fio
+engine_module=cmdline
+engine_module_parameter_base=--filename=/dev/nvme0n1 --direct=1 --rw=randread --bs=4k --name TOBEREMOVED --ioengine=libaio --iodepth=256 --group_reporting --readonly --runtime=30 --time_based --time_based --numjobs=10 --name=plop --name _REMOVE_ME
```

Comment:
FAR: the fio engine will rapidly need a "preparation" phase for storage. Why? At list this use-case: before benchmarking a flash drive should be entirely blkdiscarded then all blocks should be written once to avoid fw optimized behavior and really mean something. Since this phase would be orthogonal to hwbench's deterministic bench marking running mode we can't just use the fio engine as it is to do so. If this is of interest i'll write a feature request.

Reply:
Ack